app.json

```diff
@@ -61,13 +73,14 @@
   ],
   "buildpacks": [
     {
-      "url": "heroku/ruby"
+      "url": "heroku/nodejs"
     },
     {
-      "url": "heroku/nodejs"
+      "url": "heroku/ruby"
     },
     {
       "url": "https://github.com/heroku/heroku-buildpack-activestorage-preview"
     }
-  ]
+  ],
+  "stack": "heroku-18"
```

Comment:
So maybe we bump this in a separate PR, as discussed (some time before April 2023, I guess 😉)

Reply:
Yes, it's bumped in #390! But I was testing Heroku deploy on this PR separately and this was necessary to get it working.

https://github.com/performant-software/dm-2/pull/390/files#diff-96677aa35debfefd031d9d34d9c70369754ee3acb2d9a9d4090e98612efee6f5R70